zebrad/src/components/inbound/tests.rs

```diff
@@ -0,0 +1,72 @@
+use tower::ServiceExt;
+
+use super::mempool::{unmined_transactions_in_blocks, Mempool};
+
+use tokio::sync::oneshot;
+use tower::builder::ServiceBuilder;
+
+use zebra_chain::{
+    parameters::Network,
+    transaction::{UnminedTx, UnminedTxId},
+};
+use zebra_consensus::Config;
+use zebra_network::{Request, Response};
+
+#[test]
+fn mempool_requests_for_transaction_ids() {
+    let runtime = tokio::runtime::Builder::new_current_thread()
+        .enable_all()
+        .build()
+        .expect("Failed to create Tokio runtime");
+    let _guard = runtime.enter();
```

Comment:
This boilerplate is only necessary when the test function is inside `proptest!`, because that macro changes some things about the test function. You can simplify it here:

```suggestion
#[tokio::test]
async fn mempool_requests_for_transaction_ids() {
```

And the `runtime.block_on(async move {` can also be removed since the test function is now `async`.

Reply:
Thanks! I did this at https://github.com/ZcashFoundation/zebra/pull/2720/commits/5d07f1a4c5eb9f861d5aaa3099c0c8a62c35c7b6